File: official/atdd.extension.coder.train-interlocking/implementations/interlocking-infrastructure/fixtures/clean/python/match/wagon.py
"""match wagon — CLEAN fixture.

A wagon transforms Cargo through run_train(). It reads and returns artifact dictionaries and does NOT
import interlocking code (Cargo boundary, core afokapu/atdd#1251).
"""


def run_match(cargo):
    cargo["match"] = {"result": {"valid_for_ranked": True}}
    return cargo

File: official/atdd.workspace.bun/implementations/bun_interlocking_infrastructure/fixtures/clean/interlocking_runner_exists/src/match/wagon.ts
// match wagon — CLEAN fixture. A wagon transforms artifacts. It reads and returns a data map and
// does NOT import interlocking code (Cargo boundary, core afokapu/atdd#1251).
export function runMatch(cargo: Map<string, unknown>): Map<string, unknown> {
  cargo.set("match", { result: { validForRanked: true } });
  return cargo;
}
